connectors/sources/tests/test_mssql.py

```diff
@@ -0,0 +1,87 @@
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License 2.0;
+# you may not use this file except in compliance with the Elastic License 2.0.
+#
+"""Tests the microsoft sql database source class methods"""
+from unittest.mock import patch
+
+import pytest
+
+from connectors.sources.mssql import MSSQLDataSource, MSSQLQueries
+from connectors.sources.tests.support import create_source
+from connectors.sources.tests.test_generic_database import ConnectionSync
+
+MSSQL_CONNECTION_STRING = "mssql+pyodbc://admin:Password_123@127.0.0.1:9090/xe?TrustServerCertificate=yes&driver=ODBC+Driver+18+for+SQL+Server"
+
+
+class MockEngine:
+    """This Class create mock engine for mssql dialect"""
+
+    def connect(self):
+        """Make a connection
+
+        Returns:
+            connection: Instance of ConnectionSync
+        """
+        return ConnectionSync(MSSQLQueries())
+
+
+@patch("connectors.sources.mssql.create_engine")
+@patch("connectors.sources.mssql.URL.create")
+def test_create_engine(mock_url, mock_fun):
```

Comment:
```suggestion
def test_create_engine(mock_create_url, mock_create_engine):
```

I would be a bit more explicit here about what's mocked.

Reply:
Done